sources/refund.move

```diff
@@ -117,6 +129,71 @@ module refund::refund {
         vector::destroy_empty(amounts);
     }
 
+    // === Phase 1: Timeout setup ===
+
+    public entry fun set_timeout_ts(
```

Comment:
For simplicity, could also set this in `start_funding_phase` (add `timeout_ts` as an arg). This would let you eliminate a phase + this function.

Reply:
Implemented